userspace/libsinsp/examples/util.cpp

```diff
@@ -133,6 +133,7 @@ std::string get_event_type(uint16_t type)
         case PPME_SYSCALL_OPENAT_2_E:
         case PPME_SYSCALL_OPENAT_X:
         case PPME_SYSCALL_OPENAT_2_X: return "openat";
+        case PPME_SYSCALL_OPENAT2_X: return "openat2";
```

Comment:
We probably want the enter event here too, right?

```suggestion
        case PPME_SYSCALL_OPENAT2_E:
        case PPME_SYSCALL_OPENAT2_X: return "openat2";
```